grbl:
  ros__parameters:
    baudrate: 115200
    jog_speed: 5000.0
    max_travel_a: 150.0
    max_travel_b: 0.0
    max_travel_x: 280.0
    max_travel_y: 280.0
    max_travel_z: 65.0
    port: "/dev/ttyACM0"
    # SICK profiler scanning parameters
    # Mode: 0 - Rectangular, 1 - Circle
    scan_mode: 0
    scan_height: 100.0
    scan_width: 100.0